WinUIGallery/WinUIGallery.DesktopWap.csproj

```diff
@@ -86,5 +86,9 @@
     <Content Remove="@(Content)" />
   </ItemGroup>
   <Import Project="ContentIncludes.props" />
+  <ItemGroup>
```

Comment:
Why are you removing these? This shouldn't be necessary.

Reply:
You're right. This got added automatically by Visual Studio. Removing this now.